themes/default/content/blog/slack-integration/index.md

```diff
@@ -0,0 +1,399 @@
+
+You can either create your own Slack app (or use an existing one you may already have installed in your workspace) or you can use the link below to quickly get started with a predefined Slack app manifest.
+
+[Create a Slack app from manifest](https://api.slack.com/apps?new_app=1&manifest_yaml=display_information%3A%0A%20%20name%3A%20pulumi-slack-notifications%0A%20%20description%3A%20Funnel%20Pulumi%20webhooks%20to%20Slack%0A%20%20background_color%3A%20%22%238a3391%22%0Afeatures%3A%0A%20%20bot_user%3A%0A%20%20%20%20display_name%3A%20pulumi-slack-notifications%0A%20%20%20%20always_online%3A%20false%0Aoauth_config%3A%0A%20%20scopes%3A%0A%20%20%20%20bot%3A%0A%20%20%20%20%20%20-%20incoming-webhook%0Asettings%3A%0A%20%20org_deploy_enabled%3A%20false%0A%20%20socket_mode_enabled%3A%20false%0A%20%20token_rotation_enabled%3A%20false)
+
+By following these steps, which can also be found in [our webhooks documentation](/docs/pulumi-cloud/webhooks), in a few clicks you will have a Slack incoming webhook URL which you can use to set up a webhook in [Pulumi Cloud](https://app.pulumi.com), as shown in the GIF below.
+
+![Setting it up in the UI](webhooks_2.gif)
+
+There are Pulumi customers with hundreds to thousands of stacks, making setting up webhooks for each via the UI a time consuming task. In order to enable Pulumi notifications at scale, we have added Pulumi Cloud REST API endpoints for creating webhooks as well as [Pulumi Service Provider](https://www.pulumi.com/registry/packages/pulumiservice) support.
+
+### Pulumi Cloud REST API
+
+Create a webhook via the Pulumi Cloud REST API formatted for Slack, as shown in the example below.
+
+```bash
+
+curl \
+  -H "Accept: application/vnd.pulumi+8" \
+  -H "Content-Type: application/json" \
+  -H "Authorization: token $PULUMI_ACCESS_TOKEN" \
+  --request POST \
+  --data '{
+      "organizationName":"{organization}",
+      "projectName":"{project}",
+      "stackName":"{stack}",
+      "displayName":"#some-slack-channel",
+      "payloadUrl":"https://hooks.slack.com/services/...",
+      "format": "slack",
+      "active":true
+  }' \
+  https://api.pulumi.com/api/orgs/{organization}/{project}/{stack}/hooks
+
+
+```
+
+### Pulumi Service Provider
+
+The Pulumi Service provider allows you to create Pulumi Cloud resources via Pulumi. You can provision and manage webhooks, including Slack-formatted webhooks, using this provider.
+
+{{< chooser language "typescript,python,go,csharp,java,yaml" >}}
```

Comment:
Using a trailing slash here will ensure the chooser uses (and sets) the user's language preferred language (otherwise it only affects the choosable items that it contains): 

```suggestion
{{< chooser language "typescript,python,go,csharp,java,yaml" />}}
```